documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -28,6 +28,10 @@ export default {
 			routes: {
 				include: ['/*'],
 				exclude: ['<all>']
+			},
+			external: [ 'fs' ],
+			alias: {
+				fs: './fs-stub.js'
 			}
```

Comment:
Is using both for the same package okay? What happens in this case?

Reply:
Looking at their docs it's essentially pointless as alias resolution happens first

Reply:
I was wondering whether I should move the samples for `external` and `alias` out of the top `example` section. I believe if people are new to SveltKit / to the adapter, they will just copy whatever example from the docs they can find to get going.

Reply:
Thinking more about this, external would be for either modules that are loaded at runtime or to omit bundling something that won't be needed at runtime.

Something like `node:buffer` would be a good example for the first case but we have another PR to externalize all `node:` prefixed modules https://github.com/sveltejs/kit/pull/10544/files .

So we just have the second case, which would be something that's only dynamically imported/required on based on some condition.